README.md

```diff
@@ -75,6 +75,11 @@ X2t need more libs for convertion to pdf. All necessary
 libs in described in `DoctRenderer.config`,
 but you can use x2t inside of documentserver for easy setup
 
+To convert via xml use a numeric code in **HEX** format.
+A list of all the codes for the formats is described in the file.
+
+[OfficeFileFormats.h](https://github.com/ONLYOFFICE/core/blob/2750b471ba3e7783e903dfac7ab34eb4cfd594fe/Common/OfficeFileFormats.h#L1)
```

Comment:
I think permalink from current master will be better suited than permalink from 7.1.0 which will be deleted after merge and this may be lost

https://github.com/ONLYOFFICE/core/blob/7c835355288b10d1e8060538ddb79c2dee8eda1d/Common/OfficeFileFormats.h#L1